Source/Core/Core/IOS/Network/Socket.h

```diff
@@ -258,5 +279,11 @@ class WiiSockMan
 
   std::unordered_map<s32, WiiSocket> WiiSockets;
   s32 errno_last;
+  std::vector<poll_command> pending_polls;
+  std::vector<poll_command> failed_polls;
+  std::vector<poll_command> finished_polls;
+  std::vector<pollfd> ufds;
```

Comment:
You should probably be consistent here and use the typedef for `pollfd_t` instead.

Besides, I don't see this being used outside the `WiiSockMan::Update` method. Why don't you move it in there as a local?

Reply:
Update is called frequently, so I wanted to avoid doing frequent allocation/deallocations.
But this made me realize I forgot to clear the container.